modules/create-policy-categories.adoc

```diff
@@ -1,18 +1,18 @@
 // Module included in the following assemblies:
 //
 // * operating/manage-security-policies.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="create-policy-categories_{context}"]
 = Creating policy categories
```

Comment:
Is there a more descriptive scenario that you can tack on to this one to differentiate from the other 2 new ones you added?